src/GitVersionTask/GenerateGitVersionInformation.cs

```diff
@@ -1,42 +1,59 @@
 namespace GitVersionTask
 {
+    using System;
     using System.IO;
     using GitVersion;
     using GitVersion.Helpers;
-    using Microsoft.Build.Framework;
 
-    public class GenerateGitVersionInformation : GitVersionTaskBase
+    public static class GenerateGitVersionInformation
     {
-        [Required]
-        public string ProjectFile { get; set; }
-
-        [Required]
-        public string IntermediateOutputPath { get; set; }
-
-        [Required]
-        public string Language { get; set; }
-
-        [Output]
-        public string GitVersionInformationFilePath { get; set; }
+        public static Output Execute(
+            Input input
+            )
```

Comment:
Seems like the `input` parameter and the ending parenthesis can be put on the same line as `Execute`, as such: `Execute(Input input)`?